specification/dns/resource-manager/Microsoft.Network/stable/2016-04-01/dns.json

```diff
@@ -892,6 +892,11 @@
           "format": "int64",
           "description": "The TTL (time-to-live) of the records in the record set."
         },
+        "fqdn": {
+          "type": "string",
```

Comment:
Adding new properties requires changing the api-version according to [Azure API guidline](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning). However, since this is for s360 compliance, it should be acceptable.